src/spectral.ts

```diff
@@ -25,9 +44,42 @@ export const createSpectral = (ruleset: string) => {
     spectral.registerFormat('json-schema-draft6', isJSONSchemaDraft6);
     spectral.registerFormat('json-schema-draft7', isJSONSchemaDraft7);
     spectral.registerFormat('json-schema-2019-09', isJSONSchemaDraft2019_09);
-    await spectral.loadRuleset(ruleset);
+
+    let normRuleSet: [string] | undefined;
+
+    if (rulesetPath.length !== 0) {
+      info(`Loading ruleset '${rulesetPath}'...`);
+      normRuleSet = [rulesetPath];
+    } else {
+      info(`Loading built-in rulesets...`);
+      normRuleSet = undefined;
+    }
```

Comment:
Replace this with an expression so that normRuleSet is constant

Reply:
@XVincentX Fixed. Do you like it better?